### PR TITLE
README: commercial support

### PR DESCRIPTION
**Pull request by Mic92**
cc @dasJ does helsinki systems also wants to be mentioned here?

### CODE REVIEW

**Comment by Mic92** [comment]
Also cc @zimbatm

**Comment by dasJ** [comment]
@ajs124 do we?

**Comment by ajs124** [comment]
Sure, why not?

**Comment by cheriimoya** [comment]
indeed

File: README.md [hunk 1]
```diff
@@ -851,3 +851,11 @@ can be used together with sops-nix.
...
   mechanism to inject secrets into configuration files in the nixos activation
   phase
+
+
+# Commercial support
+
+The contributors of sops-nix are trying their best to keep sops-nix a true
+opensource and community project and not turn in into a product.
+However if you need help setting up sops-nix in your infrastructure you can
+reach out to [numtide](https://numtide.com/contact) for commercial support.
```
Comment (zimbatm) [inline README.md:861]:
```suggestion

We are building sops-nix very much as contributors to the community and are committed to keeping it open source.

That said, many of us that are contributing to sops-nix also work for consultancies. If you want to contact one of those for paid-for support setting up sops-nix in your infrastructure you can do so here:
* [Numtide](https://numtide.com/contact)
* [Helsinki Systems](https://helsinki-systems.de/)
```

**Review by zimbatm — COMMENTED** [review]
some wording suggestions, LGTM

**Pull request merged**